frontend/src/core/MarimoApp.tsx

```diff
@@ -84,7 +84,7 @@ const Providers = memo(({ children }: PropsWithChildren) => {
   return (
     <ErrorBoundary>
       <Suspense>
-        <TooltipProvider>
+        <TooltipProvider delayDuration={400}>
```

Comment:
this was 700ms previously. we put a lot of info in our tooltips so i think its good to reveal sooner